app/src/main/java/com/example/tawazz/consts/Constants.java

```diff
@@ -0,0 +1,12 @@
+package com.example.tawazz.consts;
+
+public class Constants {
+    public static final String TAWAZZ_LOG_TAG = "TAWAZZ";
+    public static final String ICON_STORAGE_DIR = "icons";
+    public static final String USERS_DATABASE_KEY = "users";
+    public static final String ROOMS_DATABASE_KEY = "rooms";
+    public static final String USER_ID_DATABASE_KEY = "id";
+    public static final String TOUCH_ACTION_DATABASE_KEY = "touch_action";
+    public static final String WIDTH_DATABASE_KEY = "width";
```

Comment:
i think `position_x` or something like that is more fitting then width, consider it
u dont have to, width is also fine